docs/learn/documentation/versioned/api/programming-model.md

```diff
@@ -65,14 +65,13 @@ Alternatively, you can implement a [TaskApplication](javadocs/org/apache/samza/a
 {% endhighlight %}
 
 
-#### Streams and Table Descriptors
+#### Streams, Side Inputs and Table Descriptors
```

Comment:
This page is for the high level programming API and should not talk about specific features. Is there another page we can add this?

Reply:
Actually Table API below is a good place for this, can remove from here.